lib/v2/oreno3d/main.js

```diff
@@ -14,81 +14,102 @@ const sortRename = {
-                return ctx.cache.tryGet(link, async () => {
-                    // 第二页数据分析
-                    const sec_data = await get_sec_page_data(link);
-                    // 传递
-                    const raw_pic_link = sec_data.raw_pic_link;
-                    const video_name = sec_data.video_name;
-                    const authors = sec_data.authors;
-                    const origins = sec_data.origins;
-                    const characters = sec_data.characters;
-                    const tags = sec_data.tags;
-                    const desc = sec_data.desc;
-                    const iwara_link = sec_data.iwara_link;
-                    const oreno3d_link = sec_data.oreno3d_link;
-                    // 打包
-                    return {
-                        title: `${video_name} - ${authors}`,
-                        author: authors,
-                        link: oreno3d_link,
-                        category: tags.split(' '),
-                        description: art(path.join(__dirname, 'templates/description.art'), {
-                            raw_pic_link,
-                            video_name,
-                            authors,
-                            origins,
-                            characters,
-                            tags,
-                            desc,
-                            iwara_link,
-                            oreno3d_link,
-                        }),
-                    };
-                });
-            })
-            .get()
+                // 第二页数据分析
+                const sec_data = await get_sec_page_data(link);
+                // 传递
+                const raw_pic_link = sec_data.raw_pic_link;
+                const video_name = sec_data.video_name;
+                const authors = sec_data.authors;
+                const origins = sec_data.origins;
+                const characters = sec_data.characters;
+                const tags = sec_data.tags;
+                const desc = sec_data.desc;
+                const iwara_link = sec_data.iwara_link;
+                const oreno3d_link = sec_data.oreno3d_link;
+                // 打包
+                return {
+                    title: `${video_name} - ${authors}`,
+                    author:authors,
+                    link: oreno3d_link,
+                    category: tags.split(' '),
+                    description: art(path.join(__dirname, 'templates/description.art'), {
+                        raw_pic_link,
+                        video_name,
+                        authors,
+                        origins,
+                        characters,
+                        tags,
+                        desc,
+                        iwara_link,
+                        oreno3d_link
+                    }),
+                };
+
+        }).get()
     );
+    return {items,title}
+    }
+    //参数定义
+    const sort = ctx.params.sort;
+    let limit = ctx.params.maxlimit ?? 1;
```

Comment:
The parameter here should match the parameter defined in `router.js` exactly.

Reply:
I've made them matched